README.md

```diff
@@ -217,14 +206,30 @@ Install additional build dependencies:
 sudo apt-get install python-vcstool python3-colcon-common-extensions
 ```
 
-Download and build MoveIt:
+Download and build MoveIt2:
 ```bash
 mkdir -p ~/moveit2_ws/src
 cd ~/moveit2_ws/src
 git clone https://github.com/AcutronicRobotics/moveit2 -b master
 cd ..
 vcs import src < src/moveit2/moveit2.repos
-colcon build --merge-install --cmake-args -DBUILD_TESTING=FALSE
+export ROS_DISTRO=dashing
+rosdep update && rosdep install -q -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO}
+colcon build --merge-install
```

Comment:
Though these instructions don't say to use --symlink-install, it might be worth warning users that using --symlink-install is currently broken for MoveIt2.

Reply:
+1